<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yeniOgrenciGiris</class>
 <widget class="QDialog" name="yeniOgrenciGiris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yeni_ogrenci_giris</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Ekle">
       <property name="text">
        <string>EKLE</string>
       </property>
       <property name="icon">
        <iconset resource="../../kaynaklar.qrc">
         <normaloff>:/resimler/resimler/resimler</normaloff>:/resimler/resimler/resimler</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Vazgec">
       <property name="text">
        <string>VAZGEÇ</string>
       </property>
       <property name="icon">
        <iconset resource="../../kaynaklar.qrc">
         <normaloff>:/resimler/resimler/okulOtomasyonu</normaloff>:/resimler/resimler/okulOtomasyonu</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_OgrenciAdi">
             <property name="text">
              <string>Öğrenci Adı</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_OgrenciAdi"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_OgrenciSoyadi">
             <property name="text">
              <string>Öğrenci Soyadı</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_OgrenciSoyadi"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_OgrenciNumarasi">
             <property name="text">
              <string>Öğrenci Numarası</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_OgrenciNumarasi"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_OgrenciAdresi">
             <property name="text">
              <string>Öğrenci Adresi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_OgrenciAdresi"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_OgrenciAdi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>yeniOgrenciGiris</receiver>
   <slot>GorselDegisti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_OgrenciSoyadi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>yeniOgrenciGiris</receiver>
   <slot>GorselDegisti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit_OgrenciAdresi</sender>
   <signal>textChanged()</signal>
   <receiver>yeniOgrenciGiris</receiver>
   <slot>GorselDegisti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Vazgec</sender>
   <signal>clicked()</signal>
   <receiver>yeniOgrenciGiris</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>GorselDegisti()</slot>
 </slots>
</ui>
